internal/logext/writer.go

```diff
@@ -16,3 +16,18 @@ func (t Writer) Write(p []byte) (n int, err error) {
 	t.ctx.Info(string(p))
 	return len(p), nil
 }
+
+// Writer writes with log.Error
+type ErrorWriter struct {
```

Comment:
this does not look like its being used...

Reply:
the struct itself could be un-exported if that's what you're suggesting, but otherwise it's used by `NewErrWriter` below, which is in turn called in `internal/exec/exec.go` to report stderr that way.

I'm open to ideas on how/whether to report stderr that way though - I just thought it should be distinguishable from stdout and reasonably formatted (which can be difficult if it's just part of an error).

Reply:
oh, sorry, didn't notice it...